lib/widgets/components/entry/register/name_body.dart

```diff
@@ -20,7 +20,7 @@ class NameBody extends StatefulWidget {
 
 class _NameBodyState extends State<NameBody> {
   final _controller = TextEditingController();
-  String get name => _controller.text;
+  String get name => _controller.text.trim();
 
   bool _showError = false;
   String? _error;
```

Comment:
Since we are trying to eliminate nullables in the codebase with the use of either, is there a reason that we are not using that here?

Reply:
This line of code was written before Either was implemented

Reply:
I presumed as much, any reason we are not interested in updating it to using Either?

Reply:
The relevant widget takes in a `String?` for conditionally displaying the validation error. Using Either will therefore be more complex.